source/_integrations/lifx.markdown

```diff
@@ -130,15 +129,19 @@ Run an effect that does nothing, thereby stopping any other effect that might be
 | ---------------------- | ----------- |
 | `entity_id` | String or list of strings that point at `entity_id`s of lights. Use `entity_id: all` to target all.
 
+## Infrared brightness
+
```

Comment:
Should we tell them it’s a select entity to make it easier for them to find?

Reply:
On a Nightvision bulb, it'll be the only non-standard entity, so it wouldn't be that hard to find.

Reply:
![2022-09-15](https://user-images.githubusercontent.com/103232/190340237-dfff1af1-fd6c-466b-bfbc-afd9902a5392.png)

Fairly hard to miss, actually.